assembl/static2/js/app/components/navbar/navbar.jsx

```diff
@@ -223,15 +227,17 @@ export default compose(
             error: data.error,
             loading: data.loading
           },
-          logoData: null
+          logoData: null,
+          isSemanticAnalysisEnabled: null
```

Comment:
It may be better to have it at `false` if it's a boolean?

Reply:
true that